services/src/handlers/plots.rs

```diff
@@ -348,7 +360,7 @@ mod tests {
             .unwrap();
 
         let params = &[
-            ("bbox", "-180,-90,180,90"),
+            ("bbox", "0,-0.3,0.2,0"),
```

Comment:
Couldn't we have a spec in such a way, that -180 to 180 reflects in the same 6 pixels than 0 to 0.2?

Reply:
no a) our reference is 0.0 so there must be an equal count of pixels on each axis for -180, 180. b) the query is designed with 3 pixels on the y axis and 2 on the x axis. The resolution to query would be y: 60, x: 90. c) this is a query that diverges a lot from the original one. I tried to keep them as designed by the authors.